### PR TITLE
fixed review to only show 1

### PR DESCRIPTION
**Pull request by asgermunk**

### CODE REVIEW

File: conferencePlanner/conferencePlannerApi/Repositories/Interfaces/IAbstractRepo.cs [hunk 1]
```diff
@@ -25,5 +25,7 @@ public interface IAbstractRepo
                 // Manipulation: Removes from database
                 // Output: True if deleted, false if not found
                 Task<bool> DeleteAsync(int id);
-        }
+
+                Task<Abstract?> UpdateReview(int abstractId, Review review);
```
Comment (Mikkelkorsaa) [inline conferencePlanner/conferencePlannerApi/Repositories/Interfaces/IAbstractRepo.cs]: Fix nullable

File: conferencePlanner/conferencePlannerApp/Components/FinalReviewComponent.razor [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+@code {
+    [Parameter]
+    public Abstract? SelectedAbstract { get; set; }
+
+    private List<AverageCriteria> GetAverageCriteriaGrades()
+    {
+        var averageCriteriaGrades = new List<AverageCriteria>();
+
+        if (SelectedAbstract?.Reviews != null)
+        {
```
Comment (Mikkelkorsaa) [inline conferencePlanner/conferencePlannerApp/Components/FinalReviewComponent.razor:133]: Fix nullable

File: conferencePlanner/conferencePlannerApp/Components/ReviewComponent.razor [hunk 3]
```diff
@@ -4,39 +4,35 @@
     <div class="abstract-review">
         @if (SelectedAbstract?.Reviews != null && SelectedAbstract.Reviews.Any())
         {
```
Comment (Mikkelkorsaa) [inline conferencePlanner/conferencePlannerApp/Components/ReviewComponent.razor:6]: Fix nullable

File: conferencePlanner/conferencePlannerApp/Components/ReviewComponent.razor [hunk 4]
```diff
@@ -4,39 +4,35 @@
...
+                                    @onclick="() => criteria.Grade = gradeValue">
+                                        @gradeValue
+                                    </button>
+                                }
                             </div>
                         </div>
+                    }
+
+                    <div class="form-group">
+                        <label class="comment-label">Comment</label>
```
Comment (Mikkelkorsaa) [inline conferencePlanner/conferencePlannerApp/Components/ReviewComponent.razor]: "Comment" = "Kommentar"

File: conferencePlanner/conferencePlannerCore/Models/Criteria.cs [hunk 5]
```diff
@@ -9,6 +9,6 @@ namespace conferencePlannerCore.Models
     public class Criteria
     {
         public string Name { get; set; } = string.Empty;
-        public int Grade { get; set; }
+        public int? Grade { get; set; }
```
Comment (Mikkelkorsaa) [inline conferencePlanner/conferencePlannerCore/Models/Criteria.cs:12]: Er det rigtigt??

**Review by Mikkelkorsaa — CHANGES_REQUESTED** [review]
Pls fix

File: conferencePlanner/conferencePlannerCore/Models/Criteria.cs (hunk 5 above)
Comment (asgermunk) [reply conferencePlanner/conferencePlannerCore/Models/Criteria.cs:12]: ja

**Review by Mikkelkorsaa — APPROVED** [review]

**Pull request merged**